src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -23,6 +23,8 @@
 
 use std::path::PathBuf;
 
+use dataflow_types::S3KeySource;
```

Comment:
Eep! The `sql-parser` crate should not depend on `dataflow_types`. Our convention is to just inline the necessary `match` statement in the typically one place that it's required in the `sql` crate.

Reply:
Ah, that makes sense, tbh I was a little surprised to see it here. I'll remove this and clean up the dependency in a follow up pr.